iv.yaml

```diff
@@ -0,0 +1,3 @@
+lenguaje: python
+entidad:
+  - money_controller/gasto.py
```

Comment:
Questo in YAML è un array con uno singolo elemento. Ciò che si cerca è una semplice coppia chiave/valore. Non è così importante adesso, ma bisogna capire come va YAML.